app/Http/Controllers/IpmaWarningController.php

```diff
@@ -17,7 +19,14 @@ class IpmaWarningController extends Controller
      */
     public function index(): JsonResponse
```

Comment:
Use the injected `Cache` instance (`$this->cache`), instead of using the facade. See https://github.com/vostpt/api/blob/master/app/Http/Controllers/Controller.php#L26